sped_esocial/models/inherited_sped_registro.py

```diff
@@ -0,0 +1,135 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 KMEE INFORMATICA LTDA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp import api, fields, models
+from openerp.exceptions import ValidationError
+
+
+class SpedRegistro(models.Model):
+    _inherit = 'sped.registro'
+
+    sped_s3000 = fields.Many2one(
+        string='Registro de Exclusão (S-3000)',
+        comodel_name='sped.registro',
+    )
+    motivo_s3000 = fields.Text(
+        string='Motivo da Exclusão',
+    )
+    situacao_s3000 = fields.Selection(
+        string='Situação',
+        selection=[
+            ('1', 'Pendente'),
+            ('2', 'Transmitida'),
+            ('3', 'Erro(s)'),
+            ('4', 'Sucesso'),
+            ('5', 'Precisa Retificar'),
+            ('6', 'Retificado'),
+            ('7', 'Excluído'),
+        ],
+        related='sped_s3000.situacao',
+    )
+    pode_excluir = fields.Boolean(
+        string='Pode Excluir?',
+        compute='compute_pode_excluir',
+    )
+
+    @api.depends('registro')
+    def compute_pode_excluir(self):
+        for registro in self:
+            registro.pode_excluir = True if registro.registro in [
+                'S-1200', 'S-1202', 'S-1207', 'S-1210', 'S-1250', 'S-1260', 'S-1270', 'S-1280', 'S-1295', 'S-1300',
+                'S-2190', 'S-2200', 'S-2205', 'S-2206', 'S-2210', 'S-2220', 'S-2230', 'S-2240', 'S-2241', 'S-2250',
+                'S-2260', 'S-2298', 'S-2299', 'S-2300', 'S-2306', 'S-2399'] else False
```

Comment:
Poderia colocar esse vetor em uma variável..
E poderia colocar apenas:
registro.registro in lista